easybuild/easyconfigs/p/PsiCLASS/PsiCLASS-1.0.3-GCC-11.2.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = "MakeCp"
+
+name = "PsiCLASS"
+version = "1.0.3"
+
+homepage = "https://github.com/splicebox/PsiCLASS"
+description = """PsiCLASS is a reference-based transcriptome assembler for single or multiple RNA-seq samples."""
+
+toolchain = {"name": "GCC", "version": "11.2.0"}
+
+source_urls = ["https://github.com/splicebox/PsiCLASS/archive/"]
+sources = ["v%(version)s.tar.gz"]
+patches = ["PsiCLASS-samtool_as_dep.patch"]
+checksums = [
+    'c60c0f33f9a076b97ccdba68b9140a5fb6936a5c5e309d70b3d0cf558828f73d',  # v1.0.3.tar.gz
+    '4cb29e2a1e9e14dd83b2d3b63a9ef508255ffb8c992e8f45565a7666d3f5e72a',  # PsiCLASS-samtool_as_dep.patch
+]
+
+dependencies = [
+    ("SAMtools", "0.1.19"),
```

Comment:
As discussed, depending on an ancient version of SAMtools is going to be a PITA when `PsiCLASS` is needed as a dependency for a tool (like [`Finder`](https://github.com/sagnikbanerjee15/Finder)) which also depends on a whole bunch of other tools, some of which require SAMtools 1.x...

So, in this case, it may be better to just stick to the bundled `samtools-0.1.9`, and not provide it as a dependency installed via EasyBuild...

Reply:
done